eng/pipelines/templates/stages/archetype-spec-gen-sdk.yml

```diff
@@ -197,16 +201,20 @@ extends:
                 $optional_params || true # return true always to suppress exit error logging
             displayName: 'Generate SDK'
 
-          - task: Powershell@2
-            inputs:
-              filePath: $(SdkRepoDirectory)/eng/common/scripts/Save-Package-Properties.ps1
-              arguments: >
-                -ServiceDirectory $(GeneratedSDK.ServiceName)
-                -OutDirectory $(GeneratedSDK.StagedArtifactsFolder)/PackageInfo
-              pwsh: true
-              workingDirectory: $(SdkRepoDirectory)
+          - pwsh: |
+              Write-Host "Service Names: $(SpecGenSdkServiceNames)"
+              Write-Host "Staged SDK Folder: $(SpecGenSdkArtifactPath)"
+            displayName: 'View SDK Generation Runtime Variables'
+
+          - pwsh: |
+              . $(SdkRepoDirectory)/eng/common/scripts/Helpers/ApiView-Helpers.ps1
```

Comment:
Looking at the Save-PackageProperties helper it looks to just call the save packages in a loop one for each service. Is there a reason you did that as a helper function as opposed to just doing it inline here?

Reply:
The service names are created at run time so I cannot do the for loop with them here. Either way am moving away from this approach